modules/store-devtools/src/extension.ts

```diff
@@ -158,4 +175,65 @@ export class DevtoolsExtension {
   private unwrapAction(action: Action) {
     return typeof action === 'string' ? eval(`(${action})`) : action;
   }
+
+  private sanitizeActions(actions: LiftedActions) {
+    if (this.config.actionSanitizer) {
+      return Object.keys(actions).reduce(
+        (sanitizedActions, actionIdx) => {
+          const idx = Number(actionIdx);
+          sanitizedActions[idx] = this.sanitizeAction(actions[idx], idx);
+          return sanitizedActions;
+        },
+        <LiftedActions>{}
+      );
+    } else {
+      return actions;
+    }
+  }
+
+  private sanitizeAction(action: LiftedAction, actionIdx: number) {
+    return this.config.actionSanitizer
+      ? {
+          ...action,
+          action: this.config.actionSanitizer(action.action, actionIdx),
+        }
+      : action;
+  }
+
+  private sanitizeStates(states: ComputedState[]) {
```

Comment:
Pass the sanitizer as an argument instead of checking its presence from the config inside the function.